contracts/Colony.sol

```diff
@@ -119,19 +119,17 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     return colonyNetwork.addSkill(_parentSkillId, true);
   }
 
-  function addDomain(uint256 _parentSkillId) public
+  function addDomain(uint256 _parentDomainId) public
   auth
-  localSkill(_parentSkillId)
```

Comment:
We don't need it now because of the `require(_parentDomainId == 1, "colony-parent-skill-not-root");`, but worth throwing in the `domainExists` modifier here now to save us forgetting to in the future?

Reply:
Since this wasn't a breaking change for us in the future I didn't think to include it but it doesn't hurt to be future-proof.